.travis.yml

```diff
@@ -44,7 +44,7 @@ matrix:
         - TESTMODE=fast
         - COVERAGE=
         - NPY_RELAXED_STRIDES_CHECKING=1
-        - NUMPYSPEC="git+git://github.com/numpy/numpy.git@v1.9.1"
+        - NUMPYSPEC="--upgrade git+git://github.com/numpy/numpy.git@v1.9.1"
```

Comment:
@ev-br I see that you're right; I had the wrong idea about what the `--upgrade` flag does.